### DISCUSSION TITLE
Amethyst 0.20.0

### DISCUSSION THREAD

**Discussion by ianyh** [ianyh/Amethyst#1532] [Announcements]
# Notable Changes
* Toggle layouts with select-*-layout commands. Thanks, @EnakNalla!
* Introduction of command-line debugging tools.
* Fix an issue that scaled windows when moving between spaces.
* Fix a frequent crash on some types of long running application launches.
* Fix an issue with multi-display focus rotations when applications are distributed across displays.

## What's Changed
* Amethyst 0.19.0 by @ianyh in https://github.com/ianyh/Amethyst/pull/1451
* Implement more command-line debugging tools by @ianyh in https://github.com/ianyh/Amethyst/pull/1457
* Bump activesupport from 7.0.4.2 to 7.0.4.3 by @dependabot in https://github.com/ianyh/Amethyst/pull/1463
* Update Silica for scoping fix by @ianyh in https://github.com/ianyh/Amethyst/pull/1467
* feat toggle window fullscreen by @EnakNalla in https://github.com/ianyh/Amethyst/pull/1358
* Grab Silica app dealloc fix by @ianyh in https://github.com/ianyh/Amethyst/pull/1470
* Update a couple of dependencies by @ianyh in https://github.com/ianyh/Amethyst/pull/1471
* Move Troubleshooting section higher up in the readme by @ianyh in https://github.com/ianyh/Amethyst/pull/1474
* Maintain frames on space changes when not changing screens by @ianyh in https://github.com/ianyh/Amethyst/pull/1372
* Implement more robust window focus mechanism by @ianyh in https://github.com/ianyh/Amethyst/pull/1530

## New Contributors
* @EnakNalla made their first contribution in https://github.com/ianyh/Amethyst/pull/1358

**Full Changelog**: https://github.com/ianyh/Amethyst/compare/v0.19.0...v0.20.0

<hr /><em>This discussion was created from the release <a href='https://github.com/ianyh/Amethyst/releases/tag/v0.20.0'>Amethyst 0.20.0</a>.</em>

**Comment by tuananh** [comment]
full screen toggle is super nice, especially on small screen
